src/main/resources/application.conf

```diff
@@ -0,0 +1,14 @@
+// Properties for configuring an Apache DBCP BasicDataSource
+// along with properties for use with a slick Database
+// Note: this is _not_ appropriate for passing to a slick Database factory method
+dbcpDataSource = {
+  driverClassName = org.postgresql.Driver
+  url = "jdbc:postgresql://google/avram?ssl=false&socketFactoryArg=broad-avram-dev:us-central1:avram&socketFactory=com.google.cloud.sql.postgres.SocketFactory"
+  username = "avram"
+  password = ""
+  maxTotal = 20 // http://slick.lightbend.com/doc/3.2.3/database.html
+  slick = {
+    numThreads = 10 // https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing
```

Comment:
Might want to adjust these comments since we're not using HikariCP. Are `numThreads`, `queueSize` still relevant?

Reply:
These are being used to configure DBCP. See DataSource.scala. The documentation link talks about choosing values and isn't really specific to HikariCP.